lib/screens/main/main_screen.dart

```diff
@@ -0,0 +1,165 @@
+
+class MainScreen extends StatefulWidget {
+  @override
+  _MainScreenState createState() => _MainScreenState();
+}
+
+class _MainScreenState extends State<MainScreen> {
+
+  @override
+  Widget build(BuildContext context) {
+    return StoreConnector<AppState, Store<AppState>>(
+      builder: (_, state) => _content(state),
+      converter: (store) => store
+    );
+  }
+
+  Widget _content(Store<AppState> store) {
+    return DefaultTabController(
+      length: 3,
+      child: Scaffold(
+        backgroundColor: Colors.white,
+        appBar: Header(),
+        bottomNavigationBar: Material(
+            color: Color(0xffffffff),
+            child: Container(
+              decoration: BoxDecoration(
+                  border: Border(
+                      top: BorderSide(color: Color(0x44B9C1D9)))),
+              height: 84.0,
+              child: Column(
+                children: [
+                  TabBar(
+                    onTap: (int index) {
+                      store.dispatch(SwitchTabAction(index));
+                    },
+                    indicator: CircleTabIndicator(
+                        color: Colors.transparent, radius: 3),
+                    tabs: [
+                      Tab(
+                        icon: Icon(Icons.home,
+                            color:
+                            // hacky, but don't want to fight with built in tabs
+                            store.state.globalState.selectedTab == 0
+                                ? Color(0xff81B4FE)
+                                : Color(0xffB9C1D9)),
+                        iconMargin: EdgeInsets.only(bottom: 0),
+                        text: '',
+                      ),
+                      Tab(
+                        icon: Icon(Icons.inbox,
+                            color:
+                            store.state.globalState.selectedTab == 1
+                                ? Color(0xff81B4FE)
+                                : Color(0xffB9C1D9)),
+                        iconMargin: EdgeInsets.only(bottom: 0),
+                        text: '',
+                      ),
+                      Tab(
+                        icon: Icon(Icons.account_circle,
+                            color:
+                            store.state.globalState.selectedTab == 2
+                                ? Color(0xff81B4FE)
+                                : Color(0xffB9C1D9)),
+                        iconMargin: EdgeInsets.only(bottom: 0),
+                        text: '',
+                      ),
+                    ],
+                  )
+                ],
+              ),
+            )),
+        body: TabBarView(
+          children: [
+            ReleaseHealth(),
+            IssuesScreenBuilder(),
+            Settings(),
+          ],
+          physics: NeverScrollableScrollPhysics(),
+        ),
+      ),
+    );
+  }
+}
+
+class Header extends StatelessWidget with PreferredSizeWidget {
+  Header({this.store});
```

Comment:
is it optional? if not, we could simply remove the {}, as its a single param and we dont have order (of the given params) problems